front/app/containers/Admin/projects/project/inputForm/index.tsx

```diff
@@ -44,4 +58,18 @@ export const IdeaForm = () => {
   );
 };
 
+const getCurrentOrLastIdeationPhase = (phases: IPhaseData[]) => {
+  const currentPhase = getCurrentPhase(phases);
+  if (currentPhase?.attributes.participation_method === 'ideation') {
+    return currentPhase;
+  }
+  const ideationPhases = phases.filter(
+    (phase) => phase.attributes.participation_method === 'ideation'
+  );
+  if (ideationPhases.length > 0) {
+    return ideationPhases.pop();
+  }
+  return undefined;
+};
```

Comment:
Just to double check I've understood - this is the same logic you have on the BE right? :)

Reply:
Ah yes of course - I forgot! Thanks :)